README.md

```diff
@@ -1,116 +1,48 @@
 
-The JavaScript API for Office enables you to create web applications that interact with the object models in Office host applications. Your application will reference the office.js library, which is a script loader. The office.js library loads the object models applicable to the Office application running the add-in.
+## This repository
 
-<br />
+This GitHub repository is primarily used to report issues found in the Office JavaScript APIs. The NPM package associated with this repo is no longer officially supported. Your add-in should get the JavaScript library from the Office content delivery network (CDN), as described in the section [Reference Office.js from the CDN](#reference-officejs-from-the-cdn). This ensures that essential product updates are served quickly to the Office Add-ins. Given that the latest Office.js release is backward-compatible with prior releases, it's safe to update to the most recent release of the library when one is available. If you need to access the library in a secure cloud environment, download the necessary files from the CDN and bundle them with your add-in.
 
-## About the NPM package
+## Report issues
 
-The NPM package for Office.js is a copy of what gets published to the official "evergreen" Office.js CDN, at **<https://appsforoffice.microsoft.com/lib/1/hosted/office.js>**.
+If you believe you have found an issue (bug) with the Office JavaScript APIs, please visit the [issues tab](https://github.com/OfficeDev/office-js/issues) of this repo. If your issue is already reported, consider adding additional context or reproduction steps. Otherwise, select **New issue**, choose **Bug report**, and provide as much detail as possible. A member of our team will respond within 1-2 business days.
 
-The Office.js CDN contains all currently available Office.js APIs at any moment in time.
+### Feature requests
 
-Each Office.js NPM package contains only Office.js APIs available on the Office.js CDN when the NPM package version was created.
+Requests for new platform features should be raised at the [Microsoft 365 Developer Platform Tech Community site](https://aka.ms/m365dev-suggestions). Upvoted requests will be considered by our team for inclusion into the product.
 
-### Target scenarios
+### Other questions
 
-The NPM package for Office.js is intended as a way to obtain an offline copy (non-CDN) of the Office.js files, which can then be statically serve from your own site instead of using the CDN.
+Questions about developing add-ins and how to use the APIs should be raised on [Stack Overflow](https://stackoverflow.com/questions/tagged/office-js) with the "office-js" tag. There, a community of experts, which includes members of our product team, will review questions and provide assistance as they are able.
```

Comment:
I thought that officially we want to be pushing [Microsoft Q&A](https://learn.microsoft.com/en-us/answers/questions/) instead of Stack. Has that changed back to Stack?

Reply:
We are currently doing both on from our docs: https://learn.microsoft.com/en-us/office/dev/add-ins/resources/resources-links-help#developer-community-forums.

I'll add links to both resources for now.